backend/siarnaq/api/teams/views.py

```diff
@@ -173,3 +175,33 @@ def avatar(self, request, pk=None, *, episode_id):
             titan.upload_image(avatar, profile.get_avatar_path())
 
         return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+    @action(
+        detail=False,
+        methods=["post"],
+        serializer_class=None,
+        permission_classes=(IsAuthenticated, IsEpisodeAvailable),
+    )
+    def record(self, request, pk=None, *, episode_id):
+        """Retrieve the win/loss record of a team"""
+        team = request.data["id"]
+        matches = Match.objects.filter(participants__team=team, status="OK!")
```

Comment:
!! Please hide tournament matches from this query!
(cuz then someone could count the difference btwn tour matches and regular matches, and use that to spoil results completely)

Reply:
As a hint from Jerry: "all team statistics should have .filter(tournament_round__isnull=True) as the first call"